go.mod

```diff
@@ -124,7 +124,7 @@ require (
 	github.com/gin-contrib/sse v0.1.0 // indirect
 	github.com/gin-gonic/gin v1.10.0 // indirect
 	github.com/go-jose/go-jose/v3 v3.0.3 // indirect
```

Comment:
I know it is not part of the card listed here but the go-jose v3 CVE fix seems to be missing, just mentioning because the v4 update is done here so it is not lost.